src/routes/index.tsx

```diff
@@ -88,7 +89,7 @@ const Routes = (): React.ReactElement => {
         }}
       />
 
-      <Route component={Home} exact path={HOME_ROUTE} />
+      <Route component={Home} exact path={[HOME_ROUTE, SAFE_ROUTES.DASHBOARD]} />
```

Comment:
What is the meaning of `HOME_ROUTE` now if we have the `SAFE_ROUTES.DASHBOARD` ?

Reply:
They are two separate routes. These do not redirect to one or the other.

Reply:
We deleted the `HOME_ROUTE` as it is not needed anymore with the new Dashboard route.